app/views/service_providers/_form.html.erb

```diff
@@ -1,5 +1,22 @@
 <%= form.error_notification %>
 <fieldset class='usa-fieldset'>
+
+  <legend class='usa-sr-only'>Production Configuration</legend>
+  <%= form.label :admin,
```

Comment:
**question:** should this be `form.label :prod_config`? My Rails form helpers / SimpleForm is a little fuzzy, so I could be off, but it seems like it should be the attribute name?

```suggestion
  <%= form.label :prod_config,
```